ui/src/tasks/components/TasksHeader.tsx

```diff
@@ -17,16 +18,16 @@ interface Props {
   showInactive: boolean
   toggleOverlay: () => void
   showOrgDropdown?: boolean
-  showFilter?: boolean
+  isFullPage?: boolean
```

Comment:
Could be a good opportunity for an enum

Reply:
Having to use a prop for this does feel kind of weird. There has to be a better way for displaying the same header in two places with different wrapper components

Reply:
Yeah, we can just keep it in mind as we patterns emerge in the orgs pages.